integration/testdata/helm-namespace/builds.out.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
should you delete this file?

Reply:
+1, it seems builds.out.json isn't used in the integration test -> might make sense to delete

Reply:
Done! I changed the test logic to add the `--build-artifacts=builds.out.json` flag conditionally and not for all the cases.